libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/js/npm/Npm.kt

```diff
@@ -141,12 +141,12 @@ class Npm internal constructor(
     ) {
         val progressLogger = objects.newBuildOpLogger()
         execWithProgress(progressLogger, description, execOps = execOps) { execSpec ->
-            val arguments: List<String> = mutableListOf<String>().apply {
+            val arguments = buildList {
                 add("install")
                 addAll(args)
                 if (logger.isDebugEnabled) add("--verbose")
                 if (environment.ignoreScripts) add("--ignore-scripts")
-            }.filter { it.isNotEmpty() }
```

Comment:
Should we leave it there if someone will pass empty arguments in the list?
Before it was used so we can write code like this:

```kotlin
if (something) test else ""
```

Should we leave it as is?